util/tournament_bracket.py

```diff
@@ -75,6 +77,25 @@ def build_bracket_tree(bracket_depth, players):
         
     return new_node
 
+def winrate(player_1, player_2, tag_rank, head_to_head):
+    p1_index = None
+    p2_index = None
+    tag1 = player_1["tag"].lower()
+    tag2 = player_2["tag"].lower()
+    if tag1 in tag_rank:
+        p1_index = tag_rank[tag1]
+    if tag2 in tag_rank:
+        p2_index = tag_rank[tag2]
+    if not p2_index:
+        return 1.0
+    if not p1_index:
+        return 0.0
+    p1_set_count = head_to_head[p1_index][p2_index]
```

Comment:
Rename `p1_set_count` to show the fact that these are p1's win count over p2; the current name does not reflect that.